MerchantSDK/GiniMerchantSDKExample/GiniMerchantSDKExample/Sources/DebugMenuViewController.swift

```diff
@@ -0,0 +1,120 @@
+//
+//  DebugMenuViewController.swift
+//
+//  Copyright © 2024 Gini GmbH. All rights reserved.
+//
+
+import UIKit
+
+protocol DebugMenuDelegate: AnyObject {
+    func didChangeReviewScreenSwitchValue(isOn: Bool)
+    func didChangeAmountEditableSwitchValue(isOn: Bool)
+}
+
+class DebugMenuViewController: UIViewController {
+    private let spacing = 20.0
+    private let rowHeight = 50.0
+
+    private lazy var titleLabel: UILabel = {
+        let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.text = "Gini Merchant"
+        label.textAlignment = .center
+        label.font = .preferredFont(forTextStyle: .largeTitle)
+        return label
+    }()
+
+    private lazy var reviewScreenOptionLabel: UILabel = rowTitle("Show Review Screen")
+
+    private lazy var reviewScreenSwitch: UISwitch = {
+        let mySwitch = UISwitch()
+        mySwitch.translatesAutoresizingMaskIntoConstraints = false
+        mySwitch.addTarget(self, action: #selector(switchValueChanged(_:)), for: .valueChanged)
+        mySwitch.isOn = showReviewScreen
+        return mySwitch
+    }()
+
+    private lazy var reviewScreenRow: UIStackView = stackView(axis: .horizontal, subviews: [reviewScreenOptionLabel, reviewScreenSwitch])
+
+    private lazy var amountEditableOptionLabel: UILabel = rowTitle("Amount field is editable")
+
+    private lazy var amountEditableSwitch: UISwitch = {
+        let mySwitch = UISwitch()
+        mySwitch.translatesAutoresizingMaskIntoConstraints = false
+        mySwitch.addTarget(self, action: #selector(switchValueChanged(_:)), for: .valueChanged)
+        mySwitch.isOn = isAmountFieldEditable
+        return mySwitch
+    }()
+
+    private lazy var amountEditableRow: UIStackView = stackView(axis: .horizontal, subviews: [amountEditableOptionLabel, amountEditableSwitch])
+
+    weak var delegate: DebugMenuDelegate?
+    private var showReviewScreen: Bool
+    private var isAmountFieldEditable: Bool
+
+    init(showReviewScreen: Bool, isAmountFieldEditable: Bool) {
+        self.showReviewScreen = showReviewScreen
```

Comment:
IDEA (not sure that good :) ) seems to this var needed only for lazy init switch - we can create switches in init and set their values, in that case we don't need to store  showReviewScreen and isAmountFieldEditable

Reply:
Or maybe create func update( with currentConfig: GiniMerchantConfig) and update all UI